apertium-eng.eng.dix

```diff
@@ -44623,60 +44627,252 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+<e lm="CS">            <i>CS</i><par n="e.g.__abbr"/></e>
+<e lm="Cu">            <i>Cu</i><par n="e.g.__abbr"/></e>
+<e lm="Cyn.">            <i>Cyn.</i><par n="e.g.__abbr"/></e>
 <e lm="d.o.b.">          <i>d.o.b.</i><par n="e.g.__abbr"/></e>
+<e lm="Db">            <i>Db</i><par n="e.g.__abbr"/></e>
 <e lm="Dec.">            <i>Dec.</i><par n="e.g.__abbr"/></e>
+<e lm="dept.">            <i>dept.</i><par n="e.g.__abbr"/></e>
+<e lm="DIY">            <i>DIY</i><par n="e.g.__abbr"/></e>
+<e lm="Dr.">            <i>Dr.</i><par n="e.g.__abbr"/></e>
+<e lm="Dy">            <i>Dy</i><par n="e.g.__abbr"/></e>
+<e lm="EA">            <i>EA</i><par n="e.g.__abbr"/></e>
 <e lm="e. g.">           <i>e.<b/>g.</i><par n="e.g.__abbr"/></e>
 <e lm="e.g.">            <i>e.g.</i><par n="e.g.__abbr"/></e>
 <e lm="ed.">             <i>ed.</i><par n="e.g.__abbr"/></e>
+<e lm="ED">            <i>ED</i><par n="e.g.__abbr"/></e>
+<e lm="ep.">            <i>ep.</i><par n="e.g.__abbr"/></e>
+<e lm="Er">            <i>Er</i><par n="e.g.__abbr"/></e>
+<e lm="ER">            <i>ER</i><par n="e.g.__abbr"/></e>
+<e lm="Es">            <i>Es</i><par n="e.g.__abbr"/></e>
 <e lm="esp.">            <i>esp.</i><par n="e.g.__abbr"/></e>
+<e lm="est.">            <i>est.</i><par n="e.g.__abbr"/></e>
 <e lm="estab.">          <i>estab.</i><par n="e.g.__abbr"/></e>
+<e lm="eta">            <i>eta</i><par n="e.g.__abbr"/></e>
 <e lm="etc.">            <i>etc.</i><par n="e.g.__abbr"/></e>
+<e lm="Eu">            <i>Eu</i><par n="e.g.__abbr"/></e>
+<e lm="EU">            <i>EU</i><par n="e.g.__abbr"/></e>
 <e lm="Ex.">             <i>Ex.</i><par n="e.g.__abbr"/></e>
 <e lm="ex.">             <i>ex.</i><par n="e.g.__abbr"/></e>
 <e lm="excl.">           <i>excl.</i><par n="e.g.__abbr"/></e>
+<e lm="F">            <i>F</i><par n="e.g.__abbr"/></e>
 <e lm="f.">              <i>f.</i><par n="e.g.__abbr"/></e>
+<e lm="Fe">            <i>Fe</i><par n="e.g.__abbr"/></e>
 <e lm="Feb.">            <i>Feb.</i><par n="e.g.__abbr"/></e>
 <e lm="fem.">            <i>fem.</i><par n="e.g.__abbr"/></e>
+<e lm="Fm">            <i>Fm</i><par n="e.g.__abbr"/></e>
+<e lm="Fr">            <i>Fr</i><par n="e.g.__abbr"/></e>
+<e lm="ft">            <i>ft</i><par n="e.g.__abbr"/></e>
+<e lm="FYI">            <i>FYI</i><par n="e.g.__abbr"/></e>
+<e lm="g">            <i>g</i><par n="e.g.__abbr"/></e>
+<e lm="Ga">            <i>Ga</i><par n="e.g.__abbr"/></e>
+<e lm="gal">            <i>gal</i><par n="e.g.__abbr"/></e>
+<e lm="Gd">            <i>Gd</i><par n="e.g.__abbr"/></e>
+<e lm="GDP">            <i>GDP</i><par n="e.g.__abbr"/></e>
+<e lm="Ge">            <i>Ge</i><par n="e.g.__abbr"/></e>
+<e lm="H">              <i>H</i><par n="e.g.__abbr"/></e>
+<e lm="He">              <i>He</i><par n="e.g.__abbr"/></e>
+<e lm="Hf">            <i>Hf</i><par n="e.g.__abbr"/></e>
+<e lm="Hg">            <i>Hg</i><par n="e.g.__abbr"/></e>
+<e lm="HIV">            <i>HIV</i><par n="e.g.__abbr"/></e>
```

Comment:
AIDS, HIV, DIY, and LSD are already in the dictionary. Others may also be but I haven't checked

Reply:
I removed some other repetitive words.